Dfe.Data.SearchPrototype/SearchForEstablishments/ByKeyword/ServiceAdapters/SearchServiceAdapterRequest.cs

```diff
@@ -13,6 +13,11 @@ public sealed class SearchServiceAdapterRequest
     /// </summary>
     public string SearchKeyword { get; }
 
+    /// <summary>
+    /// The value used to define how many records are skipped in the search response (if any).
+    /// </summary>
+    public int Offset { get; }
```

Comment:
Should this have an _explicit_ default value? 

e.g., if there is no offset given, the `int` will default to `0` but this is easy to gloss over.

Reply:
Change made. On the point above, this has already been noted and the pagination has been built with the expectation that the ticket to handle ordering will follow which such ensure such anomalies don't occur 👍

Reply:
... cog search attaches a significance to each result and returns in that order though, so it should be consistent? https://learn.microsoft.com/en-us/azure/search/search-pagination-page-layout#paging-results